ls1intum/Artemis
# Make Short Answer Options of Quiz Editor Case-Sensitive

### Checklist

### Motivation and Context
This resolves #3028 by making all options in short answer quiz questions case-sensitive, so that options that only differ in letter case are not regarded as duplicates.

### Description
- Removed `toLowerCase` on both sides.
- Make changes to saved quizzes (also in general) case-sensitive. Current problem:
   ![quiz-char-case-change](https://user-images.githubusercontent.com/11130248/111356875-11ab1b80-8689-11eb-9431-48032a2e7bac.gif)


### Steps for Testing
1. Create a quiz with short answer question
2. Add options to the short answer question that only differ in case
3. Make sure you can save the quiz (given that all unrelated problems are resolved)

### Test Coverage
_Unchanged._

### Screenshots
TODO
